.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
```

Comment:
Look at the GitHub releases and make current pr description similar to them

Reply:
@coderabbitai look at the GitHub releases and make current pr description similar to them

Reply:
@coderabbitai look at the GitHub release 1.18.0 and make current pr description similar to them

Reply:
Include them in the pr description and make the description content end-user friendly